modules/editor/init.zsh

```diff
@@ -368,6 +376,9 @@ for keymap in 'emacs' 'viins'; do
 
   # Insert 'sudo ' at the beginning of the line.
   bindkey -M "$keymap" "$key_info[Control]X$key_info[Control]S" prepend-sudo
+
+  # control-space expands all aliases, including global
+  bindkey -M "$keymap" "$key_info[Control] " globalias
```

Comment:
Should this be glob-alias, similar to above?

Reply:
My bad. I fixed it.